contracts/PolygonZkEVMBridge.sol

```diff
@@ -153,6 +155,9 @@ contract PolygonZkEVMBridge is
             // Ether is treated as ether from mainnet
             originNetwork = MAINNET_NETWORK_ID;
         } else {
+            // Check msg.value is 0 if tokens are bridged
+            require(msg.value == 0, "PolygonZkEVMBridge::bridgeAsset: Expected zero native asset value when bridging ERC20 tokens");
```

Comment:
What if there is an ERC20 token that requires some `msg.value` in order to perform its `safeTransferFrom` custom implementation?

Reply:
This is not contemplated, and if it was the case we should send the msg.value along with the transferFrom function, which i think by usually is not payable

Reply:
This avoids bad usage of the Bridge but I think it is not required xD